packages/amplify-category-function/provider-utils/awscloudformation/utils/cron.js

```diff
@@ -0,0 +1,267 @@
+                    Ref: 'LambdaFunction',
+                  },
+                },
+              ],
+            },
+          };
+          // append permissions to invoke lambda via cloiudwatch to CFN file
+          cfnContent.Resources.PermissionForEventsToInvokeLambda = {
+            Type: 'AWS::Lambda::Permission',
+            Properties: {
+              FunctionName: {
+                Ref: 'LambdaFunction',
+              },
+              Action: 'lambda:InvokeFunction',
+              Principal: 'events.amazonaws.com',
+              SourceArn: { 'Fn::GetAtt': ['CloudwatchEvent', 'Arn'] },
+            },
+          };
+          fs.writeFileSync(cfnFilePath, JSON.stringify(cfnContent, null, 4));
+        }
+      } catch (e) {
+        context.print.error(e.message);
+      }
+    }
+  } else {
+    if (await context.amplify.confirmPrompt.run('Do you want to Update/Remove the ScheduleEvent Rule?', false)) {
+      const cfnContent = context.amplify.readJsonFile(cfnFilePath);
+      const scheduleEventOperationQuestion = {
+        type: 'list',
+        name: 'ScheduleEventOperation',
+        message: 'Select from the following options',
+        choices: ['Update the CronJob', 'Remove the CronJob'],
+      };
+
+      const scheduleEventOperationAnswer = await inquirer.prompt([scheduleEventOperationQuestion]);
+
+      switch (scheduleEventOperationAnswer.ScheduleEventOperation) {
+        case 'Update the CronJob': {
+          // add service walkthrough to get the cron expression
+          cloudWatchRule = await cronServiceWalkthrough(context, parameters);
+          parameters.cloudwatchEvent = 'true';
+          parameters.cloudwatchRule = cloudWatchRule;
+          cfnContent.Resources.CloudWatchEvent.Properties.ScheduleExpression = parameters.cloudwatchRule;
+          fs.writeFileSync(cfnFilePath, JSON.stringify(cfnContent, null, 4));
+          break;
+        }
+        case 'Remove the CronJob': {
+          parameters.cloudwatchEvent = 'NONE';
+          parameters.cloudwatchRule = 'NONE';
+          delete cfnContent.Resources.CloudWatchEvent;
+          delete cfnContent.Resources.PermissionForEventsToInvokeLambda;
+          fs.writeFileSync(cfnFilePath, JSON.stringify(cfnContent, null, 4));
+          break;
+        }
+        default:
+          console.log(`${scheduleEventOperationAnswer.scheduleEventOperation} not supported`);
+      }
+    }
+  }
+}
+
+async function cronServiceWalkthrough(context, parameters) {
+  let cloudwatchRule;
+  // resource questions for setting cron
+  const timeQuestion = {
+    type: 'datetime',
+    name: 'dt',
+    message: 'When would you like to start cron?',
+    format: ['h', ':', 'MM', ' ', 'TT'],
+  };
+  const timeAnswer = await inquirer.prompt([timeQuestion]);
+
+  const intervalQuestion = {
+    type: 'list',
+    name: 'interval',
+    message: 'Please select interval or select custom rule exp?',
+    choices: ['daily', 'weekly', 'monthly', 'yearly', 'customRule'],
```

Comment:
We need to have minutes and hours as well to support like "every hour", "every 5 minutes".

Reply:
Included every hour and every minute functionality